From 6002aac705d8987b79226df815a884cb2ad67116 Mon Sep 17 00:00:00 2001
From: Jonas Albrecht <plonkbong100@protonmail.com>
Date: Mon, 11 Apr 2022 18:00:15 +0000
Subject: [PATCH] fix: Makefile for make defconfig

add possibility to configure the source via
make MINICONFIG="--olddefconfig" defconfig

Signed-off-by: Jonas Albrecht <plonkbong100@protonmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f9dd0f9..caf5ad2 100755
--- a/Makefile
+++ b/Makefile
@@ -143,7 +143,7 @@ MINICONFIG:=/dev/null
 endif
 
 .config_d:
-	./tools/kconfig/conf -D $(MINICONFIG) MTLKConfig
+	./tools/kconfig/conf -s $(MINICONFIG) MTLKConfig
 	test .config.h -nt .config || rm -f .config.h
 
 %: 
-- 
2.32.0

